### DISCUSSION TITLE
Use filter-param in ItemsService

### DISCUSSION THREAD

**Discussion by guidoka** [directus/directus#8572] [Archive]
Hi, up to now i used rc85 - this statement worked

        const syncItems = await syncService.readByQuery({
            fields: [*],
            filter: {
                'processed' : { '_eq': null },
                'source' : { '_eq': 1 },
            },
            sort: [{column:'date_created', order: 'asc'}],
        });

Result should be all Records with processed timestamp equals null and source equals 1.
I updated to 9.0.0-rc.95 (also database migrated:latest).

Now this statement returns zero rows. Is there a change in Filter-handling. I didn't find anything in the documentation

**Comment by joselcvarela** [comment]
Hi @guidoka ,
There's a lot of changes between those versions, so maybe could have been changed on filters.
My guess is you should do `'processed' : { '_null': true },` instead of `'processed' : { '_eq': null },`

**Reply by guidoka** [reply]
HI @joselcvarela, thanks a lot - that's it. I did not get that change. Sorry :-)
